src/style/Navbar.css

```diff
@@ -0,0 +1,46 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap');
+
+.navbar {
+  width: 87.5rem;
+  height: 5.938rem;
+
+  /* margin: 0 0 2.375rem; */
+  padding: 1.438rem 1.188rem;
+}
+
+.navbar-brand {
+  margin: 0;
+}
+
+.navbar__items {
+  display: flex;
+}
+
+.bookstore__name {
+  width: 15rem;
+  height: 2.313rem;
+  margin: 0.375rem 2.938rem 0.125rem 0;
+  font-family: Montserrat, 'Times New Roman', Times, serif;
```

Comment:
- [ ] It would be better if we implement the CSS DRY principle, these rules have been repeated. Let us kindly fix them